test/integration/models/linode/test_linode.py

```diff
@@ -512,21 +512,27 @@ def test_linode_ips(create_linode):
     assert ips.ipv4.public[0].address == linode.ipv4[0]
 
 
-def test_linode_initate_migration(test_linode_client):
+# TODO(LDE): Remove skip once LDE is available
+@pytest.mark.skip("LDE is not currently enabled")
+def test_linode_initate_migration(test_linode_client, e2e_test_firewall):
```

Comment:
Just curious, are migrations of encrypted Linodes supported yet or are we just not targeting regions that support LDE?

Reply:
I am not sure regarding this.. I tried with different regions but same error. This test used to pass according to TOD, I will try to get an update regarding this.

Reply:
Nice, thanks @ykim-1!